fetch.bs

```diff
@@ -734,7 +734,7 @@ requests, but not service worker requests. It references an
 <a>environment settings object</a> for a worker request.
 
 <p>A <a for=/>request</a> has an associated
-<dfn id=concept-request-target-client-id export for=request>target client id</dfn>
+<dfn id=concept-request-replaces-client-id export for=request>replaces client id</dfn>
 (a string). Unless stated otherwise it is the empty string.
 
 <p class="note no-backref">This is only used by <a>navigation requests</a>. It is the
```

Comment:
Is this note still accurate?

Reply:
Yes, no semantical change, just naming change.